source/aggregation-tutorials/one-to-one-join.txt

```diff
@@ -20,69 +20,79 @@ One-to-One Join
 Introduction
 ------------
 
-In this tutorial, you will learn how to perform a one-to-one join using
-aggregation with the {+driver-long+}. A one-to-one join occurs when a document
-in one collection has a field value that matches to a single document with the
-same field value in another collection. The two documents with matching fields
-are then combined into one document.
+In this tutorial, you can learn how to use the {+driver-short+} to
+perform a one-to-one join. A one-to-one join occurs when a document
+in one collection has a field value that matches a single document in
+another collection with the same field value. The aggregation matches
+these documents on the field value and combines information from both
+into one document.
 
 .. tip::
 
-   A one-to-one join does not require the entities to have a :wikipedia:`One-to-one relationship
-   </One-to-one_(data_model)>`.
+   A one-to-one join does not require the documents to have a
+   One-to-one relationship. To learn more about this data relationship,
```

Comment:
I don't think capitalization is necessary here - only in reference to the title of the linked doc.
```suggestion
   one-to-one relationship. To learn more about this data relationship,
```